_data/translation-management-systems.yaml

```diff
@@ -0,0 +1,669 @@
+---
+#- id: [Lowercase, words separated by hyphen]
+#  name:
+#  description:
+#  url:
+#  cat: [Boolean]
+#  documentation: [Link]
+#  machine_translation_integrations:
+#    - service: [Use the corresponding engine id]
+#      custom: [Is the engine custom model available from the tool?]
+#      tone: [Is tone/formality selection available from the tool?]
+#      glossary: [Is the machine translation glossary/terminology/dictionary available from the tool?]
+#      adaptive: [Is the adaptive machine translation feature available from the tool?]
+#
+#  machine_translation_plugins:
+#    - service:
+#      custom:
+#      glossary:
+#      tone:
+#      adaptive:
+#
+#  features:
+#    - name:
+#      description:
+
+- id: crowdin
+  name: crowdin
+  description: Translation management system
+  url: https://www.crowdin.com
+  cat: true
+  documentation: https://support.crowdin.com/configuring-machine-translation-engines/
+  machine_translation_integrations:
```

Comment:
Crowdin really only supports custom for Google and Microsoft, and tone for DeepL.

For anything else, you need to write your own custom integration app (or use Intento).

<img width="439" alt="Screenshot 2022-07-09 at 10 44 04" src="https://user-images.githubusercontent.com/11457984/178094942-7dcacf40-1d03-4429-9910-caab90d13520.png">